src/schema/slack/types/mod.ts

```diff
@@ -0,0 +1,10 @@
+const SlackTypes = {
```

Comment:
It might be a little confusing that this is called `SlackTypes` here, when throughout our code base we import `SlackTypes` from `src/schema/slack/schema_types.ts` which will be the combination of these slack primitives and the slack custom types. The only case where we import this is in `src/parameters.ts/types.ts`